cmd/operator/register/root.go

```diff
@@ -116,7 +118,8 @@ func run(config cfg.Config) {
 	registration := config.Elemental.Registration
 
 	if registration.URL == "" {
-		return
+		logrus.Fatal("Registration URL is empty")
+		os.Exit(1)
```

Comment:
I think the os.Exit(1) is not needed, as far as I understand the `logrus.Fatal` already exits 1. https://pkg.go.dev/github.com/sirupsen/logrus#Fatal

Reply:
oh good catch!